backend/src/appointment/routes/subscriber.py

```diff
@@ -27,7 +27,7 @@ def disable_subscriber(
     email: str, db: Session = Depends(get_db), subscriber: Subscriber = Depends(get_admin_subscriber)
 ):
     """endpoint to mark a subscriber deleted by email, needs admin permissions"""
-    subscriber_to_delete = repo.subscriber.get_by_email(db, email)
+    subscriber_to_delete = repo.subscriber.get_by_email(db, email.lower())
```

Comment:
Since we lower inside the `get_by_email()` function now, we can skip that here, right?